.github/workflows/release-nethermind.yml

```diff
@@ -38,20 +38,29 @@ jobs:
     - name: Setting up Node.js
       uses: actions/setup-node@master
       with:
-        node-version: "13"
+        node-version: "14"
     - name: Setting up Build Environment
       run: |
         npm i pkg @vercel/ncc -g
+    - name: Setting up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Setting up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
     - name: Setting up Packages
       run: ./nethermind/scripts/deployment/setup-packages.sh
     - name: Building Runner
       run: ./nethermind/scripts/deployment/build-runner.sh
-    - name: Building BeamWallet
-      run: ./nethermind/scripts/deployment/build-wallet.sh
     - name: Building Cli
       run: ./nethermind/scripts/deployment/build-cli.sh
     - name: Building Launcher
       run: ./nethermind/scripts/deployment/build-launcher.sh
+    - name: Building Launcher for arm64
+      run: |
+        cd nethermind/
+        docker buildx build --platform=linux/arm64 -t tmp-launcher -f Dockerfile.launcher . --load
+        docker run -v $PWD:/opt/mount --rm tmp-launcher bash -c "cp /nethermind/Nethermind.Launcher /opt/mount/"
+        mv Nethermind.Launcher ${{ env.RELEASE_DIRECTORY }}/nethermind-lin-arm64/Nethermind.Launcher
```

Comment:
Can we use the environment variable instead of "nethermind-lin-arm64"
Just to maintain a common standard across the workflows.